src/core/cgroup.c

```diff
@@ -3989,6 +3989,38 @@ int unit_get_ip_accounting(
         return r;
 }
 
+static uint64_t unit_get_eff_limit_one(Unit *u, CGroupLimitType type) {
+        CGroupContext *cc;
+
+        assert(u);
+        assert(UNIT_HAS_CGROUP_CONTEXT(u));
+
+        static const size_t members[_CGROUP_LIMIT_MAX] = {
+                [CGROUP_LIMIT_MEMORY_MAX]    = offsetof(CGroupContext, memory_max),
+                [CGROUP_LIMIT_MEMORY_HIGH]   = offsetof(CGroupContext, memory_high),
+                [CGROUP_LIMIT_TASKS_MAX]     = offsetof(CGroupContext, tasks_max),
```

Comment:
aligning this as a table is good, but please only one space as separator of the columns

Reply:
fixed.